src/index.js

```diff
@@ -154,6 +156,9 @@ export default class Sound extends React.Component {
       whileplaying() {
         props.onPlaying(this);
       },
+      onerror() {
```

Comment:
This event receives some useful parameters, such as the error number 
 and description (http://www.schillmania.com/projects/soundmanager2/doc/#smsound-onerror). Maybe we could add them as parameters in the callback prop? we could do:
```js
props.onError(errorCode, description, this)
```

Reply:
Sweet. Sounds good. I'll get around to this later on today, then resubmit 👌